devel/raja/Portfile

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           mpi 1.0
+
+github.setup        LLNL RAJA 2022.10.5 v
+name                raja
+revision            0
+categories          devel
+license             BSD
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         RAJA Performance Portability Layer (C++)
+long_description    RAJA is a library of C++ software abstractions, primarily developed \
+                    at Lawrence Livermore National Laboratory (LLNL), that enables architecture \
+                    and programming model portability for HPC applications.
+
+fetch.type          git
+
+if {${os.major} < 14} {
+    # Lion+ (with Xcode 4.1+) have git; earlier need to bring their own.
+    # However Lion through Mavericks fail with SSL errors.
+    depends_build-append \
+                    port:git
+    git.cmd         ${prefix}/bin/git
+}
+
+set py_ver          3.11
+set py_ver_nodot    [string map {. {}} ${py_ver}]
+depends_build-append \
+                    port:pkgconfig \
+                    port:python${py_ver_nodot}
+depends_lib-append  port:cppcheck
```

Comment:
is that really a build/runtime dependency?

Reply:
```
#------------------------------------
# Static analysis via Cppcheck
#------------------------------------
blt_find_executable(NAME        Cppcheck
                    EXECUTABLES cppcheck)
```
In `SetupThirdParty.cmake`. But I do not insist on it, and do not know if it actually does something.

Reply:
Yeah looks like it is not needed, I will remove it.